layouts/shortcodes/shell.html

```diff
@@ -0,0 +1,53 @@
+<!--
+{{/*
+Short code to add a bash command line input/output block
+Usage:
+command-line [username] [hostname] [language]
+username - The username to appear in the prompt, defaults to 'user'
+hostname - The hostname to appear in the prompt, defaults to 'localhost'
+language - The language of the content, defaults to bash
+If user or host are provided then the prompt will be '[<user>@<host>] $'
+else it will just be '$'.
+Suffix lines with ' \' and the next line will be treated as a continuation
+Prefix lines with '(out)' and it will render them as output lines
+See https://prismjs.com/plugins/command-line/
+
+e.g.:
+{{% shell "jdoe" "bigserver" %}}
+echo hello \
+and \
+goodbye
+(out)hello and goodbye
+{{% /shell %}}
+*/}}
+-->
+
```

Comment:
Maybe we could add something like:

```suggestion
{{ if not (.Site.Params.prism_syntax_highlighting) }}
    {{ warnf "Warning: This shortcode requires the PrismJS highlighter, it doesn't work with Chroma. For details, see https://www.docsy.dev/docs/adding-content/lookandfeel/#code-highlighting-with-prism , %s" .Position -}}
{{ end }}
```

That would solve https://github.com/google/docsy/pull/1012#issuecomment-1152734489

Reply:
Thanks for this.  I have added it in for shell and sql-shell, though have changed it to `errorf` as warnings tend to get ignored and it is not usable without prism